python/tests/plugins/lsp_server/base_lsp_test_case.py

```diff
@@ -36,10 +36,6 @@ async def asyncSetUp(self) -> None:
         self.active_context = self.client.server.language_context
         self.temp_documents_directory: TemporaryDirectory = TemporaryDirectory()
 
-        # Add the core spec to the virtual docs
-        core_spec_virtual_doc = self._create_core_spec_virtual_doc()
-        self.documents[core_spec_virtual_doc.get_full_path()] = core_spec_virtual_doc
```

Comment:
Was this intentionally deleted? I think we want this but would want to remove adding the spec from certain tests where it's being explicitly added. For example, lines 133, 144, and 161, here:

https://github.com/jondavid-black/AaC/blob/c64b3ab31952d34739c2af3c117cffa95f91dffb/python/tests/plugins/lsp_server/providers/test_goto_definition_provider.py#L132-L161